Makefile

```diff
@@ -0,0 +1,20 @@
+TEMURIN_TAGS := 8u322-b06-jdk 11.0.14.1_1-jdk 8u322-b06-jre 11.0.14.1_1-jre
+JOBS := $(addprefix temurin-,${TEMURIN_TAGS})
+CHECKS := $(addprefix check-,${TEMURIN_TAGS})
+CLEAN := $(addprefix clean-,${TEMURIN_TAGS})
+
+all: ${JOBS}
+check: ${CHECKS}
+.PHONY: all ${JOBS}
```

Comment:
Since `check` and `${CHECKS}` don't have any outputs on the filesystem, they should also be added to this `.PHONY` list, so that make won't skip them thinking there's nothing to be done.

```make
.PHONY: all check clean ${JOBS} ${CHECKS} ${CLEAN}
```